### ISSUE TITLE
Packages are not published even though npm reports as successful

### ISSUE THREAD

**Issue by canterberry** [npm/registry-issue-archive#352]
**Impact:** Severe
**Symptom:** Unable to publish packages

Since at least **Tuesday, June 5, 2018 at 5:03:40 PM UTC-7**, our CI build tooling has been unable to successfully publish private packages to our organization on npmjs.com. The CLI output indicates that the publication was successful, but the package does not appear in npmjs.com. The issue has persisted until at least **Friday, June 8, 2018 at 12:40:17 PM UTC-7**. The symptoms appear to be the same as the issue logged on https://status.npmjs.org/ on June 1.

For assistance in diagnosing this issue, the pertinent configuration, command, and output is as follows:

**~/.npmrc**:

```
//registry.npmjs.org/:_authToken=<VALID_TOKEN>
```

**Command:**

```
$ yarn publish \
  --access restricted \
  --cwd lib \
  --new-version "$(perl -n -e'/^  "version": "(.+)",$/ && printf "$1";' < package.json)" \
  --no-git-tag-version \
  --no-progress \
  --non-interactive \
  --production
```

**Output:**

```
yarn publish v1.7.0
[1/4] Bumping version...
info Current version: 0.0.2
[2/4] Logging in...
[3/4] Publishing...
success Published.
[4/4] Revoking token...
info Not revoking login token, specified via config file.
Done in 9.97s.
```

I'm happy to provide whatever additional information necessary to help diagnose and address this issue.

**Comment by canterberry** [comment]
Just to follow up on this:

I still receive an email telling me my package is published (even though it never shows up in the registry when accessing either by CLI, via package.json, or anywhere in the registry UI). The email I receive is as follows (some details redacted for confidentiality):

```
Hi ###USER###!

A new version of the package @###ORG###/###PACKAGE### (###VERSION###) was published at 2018-06-09T20:03:47.632Z from
###IP###. The shasum of this package was 5ea…341.

If you have questions or security concerns, you can reply to this message or
email support@npmjs.com.

npm loves you.
```

As a potential workaround, I have tried setting the registry explicitly before publishing, like so:

```
$ yarn config set registry https://registry.npmjs.org
```

This resulted in the same outcome, and my package still was not published.

**Comment by canterberry** [comment]
Is there anyone monitoring this repo for new issues?
